pallets/streaming/src/types.rs

```diff
@@ -70,9 +75,13 @@ impl<T: Config> Stream<T> {
             recipient,
             start_time,
             end_time,
-            status: StreamStatus::Ongoing,
+            status: StreamStatus::Ongoing {
+                as_collateral: false,
+            },
+            cancellable: true,
```

Comment:
Should we add it to the create method and set the default true if not provided?

Reply:
If we support creating a non-cancellable stream, we need to add a new field to create(), and we must pass in field

Reply:
Already added